operator/v1/types_ingress.go

```diff
@@ -716,8 +720,61 @@ type AWSNetworkLoadBalancerParameters struct {
 	// +optional
 	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
 	Subnets *AWSSubnets `json:"subnets,omitempty"`
+
+	// eipAllocations assign Elastic IP addresses to the Network Load Balancer.
+	//
+	// The following restrictions apply:
+	//
+	// eipAllocations cannot be used with "Internal" but "External" scope.
+	//
+	// The number of Allocation IDs must match the number of subnets that are used for the load balancer.
+	//
+	// An EIP can be allocated to only a single IngressController.
+	//
+	// Each entry in the eipAllocations list must be unique.
+	//
+	// A maximum of 10 IP allocations are permitted.
+	//
+	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html for general
+	// information about configuration, characteristics, and limitations of Elastic IP addresses.
```

Comment:
*nit* I think it's a standard to separate the kubebuilder tags from the go docs with a space.

```suggestion
	// information about configuration, characteristics, and limitations of Elastic IP addresses.
	
```

Reply:
ok done